docs/usage/configuration-options.md

```diff
@@ -2246,7 +2246,30 @@ In the case that a user is automatically added as reviewer (such as Renovate App
 
 ## ignoreScripts
 
-Applicable for npm, bun, Composer and Copier only for now. Set this to `true` if running scripts causes problems.
+By default, Renovate will disable package manager scripts.
+Allowing packager manager scripts is a risk:
+
+- Untrusted or compromised repository users could use package manager scripts to exploit the system where Renovate runs, and
+- Malicious package authors could use scripts to exploit a repository and Renovate system, for example to exfiltrate source code and secrets
+
+<!-- markdownlint-disable MD001 -->
+
+#### No script execution on free Mend-hosted Renovate
```

Comment:
I'd prefer we just use **bold** as otherwise this will bloat our side menu in docs